packages/avatar/src/vaadin-avatar-mixin.d.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import type { Constructor } from '@open-wc/dedupe-mixin';
+import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin';
```

Comment:
nit: let's add `.js` extension for consistency as we generally use it.

```suggestion
import type { FocusMixinClass } from '@vaadin/a11y-base/src/focus-mixin.js';
```